menu.kando.Kando.yml

```diff
@@ -0,0 +1,40 @@
+id: menu.kando.Kando
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: AppRun
+finish-args:
+  - --share=ipc
+  - --device=dri
+  - --socket=x11
+  - --talk-name=org.freedesktop.Flatpak
+  - --talk-name=org.gnome.Shell
+  - --talk-name=org.kde.KWin
+modules:
+  - name: kando
+    buildsystem: simple
+    sources:
+      - type: file
+        dest-filename: kando.AppImage
+        url: https://github.com/kando-menu/kando/releases/download/v1.6.0/Kando-1.6.0-x86_64.AppImage
+        sha256: 4448419ab790e2623e590439b8b4202d14030e5ee1a4cd2add4286dff329e670
+      - type: file
+        dest-filename: menu.kando.Kando.svg
+        url: https://raw.githubusercontent.com/kando-menu/kando/08f8777e19206f00ab203b844c056a0f7a296f98/tools/appstream/menu.kando.Kando.svg
+        sha256: a31e870e267c78e80802648616ea18c0cdfe6a1297c91b6a7998d762326a0bb1
+      - type: file
+        dest-filename: menu.kando.Kando.desktop
+        url: https://raw.githubusercontent.com/kando-menu/kando/08f8777e19206f00ab203b844c056a0f7a296f98/tools/appstream/menu.kando.Kando.desktop
```

Comment:
you mean the appimage come without a desktop file or icon? What the point of pre-building if it can't even follow 20 year old standards for desktop applications.

Reply:
I answered your comment above.